test/staking/TokenStaking.test.js

```diff
@@ -5818,6 +6063,12 @@ describe("TokenStaking", () => {
           ])
         })
 
+        it("should decrease the delegatee voting power", async () => {
+          expect(await tokenStaking.getVotes(delegatee.address)).to.equal(
+            expectedAmount
+          )
+        })
```

Comment:
I think we should add a test that it `should decrease the total voting power` as well.